packages/ckb-sdk-core/src/signWitness.ts

```diff
@@ -0,0 +1,12 @@
+import { blake2b, hexToBytes, bytesToHex, PERSONAL } from '@nervosnetwork/ckb-sdk-utils'
+import Address from './address'
+
+export default (addrObj: Address, transactionHash: string, witness: string = '') => {
```

Comment:
`signWitness` is a action, so maybe it's not appropriate for file name.

Reply:
It's better to keep the name the same as the function exported by default in js.

Reply:
OK